### PR TITLE
Rel 508653 processing source location code update

### PR DESCRIPTION
**Pull request by ppande1020**
Added code to create processing source location and an integration test for it.

### CODE REVIEW

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 	{
 		public const string TEST_APPLICATION_NAME = "Production";
-		public const string RELATIVITY_INSTANCE_NAME = "172.24.170.148";
-		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Data Grid Workspace";
+		public const string RELATIVITY_INSTANCE_NAME = "172.17.208.218";
+		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Workspace";
 		public const string RELATIVITY_INSTANCE_WEB_SERVICE_URL = "http://" + RELATIVITY_INSTANCE_NAME + "/relativitywebapi/";
 		public const string RELATIVITY_ADMIN_USER_NAME = "relativity.admin@relativity.com";
-		public const string RELATIVITY_ADMIN_PASSWORD = "<password_removed_contact_relativity_plex_team>";
+		public const string RELATIVITY_ADMIN_PASSWORD = "Test1234!";
```
Comment (TommyBigNose) [inline CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs:10]: Might want to set this file back to default

**Review by TommyBigNose — CHANGES_REQUESTED** [review]
One file needs to be reverted

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs (hunk 1 above)
Comment (ppande1020) [reply CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs:10]: Yes. I can revert this change

Comment (ppande1020) [reply CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs]: its done. please pull latest from this branch

**Review by TommyBigNose — APPROVED** [review]

**Review by jake-martin — DISMISSED** [review]

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs [hunk 2]
```diff
@@ -3,8 +3,8 @@
 	public class TestConstants
 	{
 		public const string TEST_APPLICATION_NAME = "Production";
-		public const string RELATIVITY_INSTANCE_NAME = "172.24.170.148";
-		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Data Grid Workspace";
+		public const string RELATIVITY_INSTANCE_NAME = "172.17.208.218";
+		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Workspace";
```
Comment (calimetikcura) [inline CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs:7]: can we update the variable name to remove datagrid as its updated?

File: CSharp/DevVmPowershell/Helpers/Constants.cs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ﻿using Relativity.Imaging.Services.Interfaces;
 using Relativity.Services.Agent;
 using System;
+using kCura.Vendor.Castle.DynamicProxy.Generators.Emitters.SimpleAST;
```
Comment (calimetikcura) [inline CSharp/DevVmPowershell/Helpers/Constants.cs:4]: do we need this using statement? i don't think we need it.

File: CSharp/DevVmPowershell/Helpers/Implementations/ProcessingHelper.cs [hunk 4]
```diff
@@ -111,59 +113,49 @@ public async Task<bool> FullResetAsync()
...
+			string resultString = await createResponse.Content.ReadAsStringAsync();
 
-				}
-				else
-				{
-					Console.WriteLine($"{nameof(CreateProcessingSourceLocationChoice)} - Failed to create Processing Source Location Choice ({Constants.Processing.ChoiceName})");
-				}
+			int processingChoiceId = Int32.Parse(resultString);
+
+			if (processingChoiceId == null || processingChoiceId < 0)
```
Comment (calimetikcura) [inline CSharp/DevVmPowershell/Helpers/Implementations/ProcessingHelper.cs:151]: processingChoiceId == null check can be removed as int doesn't allow null values.

**Review by calimetikcura — CHANGES_REQUESTED** [review]

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs (hunk 2 above)
Comment (ppande1020) [reply CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs:7]: leaving it as is

**Review by calimetikcura — APPROVED** [review]

**Review by TommyBigNose — APPROVED** [review]

**Pull request merged**